### PR TITLE
1.9.3 changelog

### PR DESCRIPTION
**Pull request by jamiedemaria**

### CODE REVIEW

**Comment by jamiedemaria** [comment]
* **#26034** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26034?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26034?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `master`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,53 @@
...
+- [ui] Fixed an issue introduced in 1.9.2 where the backfill overview page would sometimes display extra assets that were targeted by the backfill.
+- [ui] Fixed "Open in Launchpad" button when testing a schedule or sensor by ensuring that it opens to the correct deployment.
+- [ui] Fixed an issue where switching a user setting was immediately saved, rather than waiting for the change to be confirmed.
+- [dagster-looker] Unions without unique/distinct criteria are now properly handled.
+- [dagster-powerbi] Fixed an issue where reports without an upstream dataset dependency would fail to translate to an asset spec.
+- [dagster-sigma] Fixed an issue where API fetches did not paginate properly.
+
+### Documentation
+
+- Added an [Airflow Federation Tutorial](https://docs.dagster.io/integrations/airlift/federation-tutorial/overview).
```
Comment (dpeng817) [inline CHANGES.md:40]: Can we just have an airlift section added to this release? This is what I'm thinking:
```
### Introducing: `dagster-airlift`

`dagster-airlift` is coming out of stealth, and we've added a bunch of airlift content to the docs site. See the initial Airlift RFC [here](https://github.com/dagster-io/dagster/discussions/25279).
- A full [Airflow migration tutorial](https://master.dagster.dagster-docs.io/integrations/airlift/tutorial/overview
- A tutorial on [federating between Airflow instances](https://master.dagster.dagster-docs.io/integrations/airlift/federation-tutorial/overview)
- A bunch of other Airflow-related content coming soon! We'd love for you to check it out, and post any comments / questions in the #airflow-migration channel in the Dagster slack.

Comment (jamiedemaria) [reply CHANGES.md:48]: did some wordsmithing to make it match the tone of the changelog more closely and nested it under the New section. I'm going to see what the preview looks like to see if it looks ok. I don't know if we've done this special kind of callout for a library before, so we might need to mess with the formatting so it looks good

**Review by dpeng817 — APPROVED** [review]
thanks!

**Pull request merged**